docs/release-notes/v0.40.0.md

```diff
@@ -0,0 +1,88 @@
+## Announcing Radius v0.40.0
+
+Today we're happy to announce the release of Radius v0.40.0. Check out the [highlights](#highlights) below, along with the [full changelog](#full-changelog) for more details.
+
+We would like to extend our thanks to all the [new](#new-contributors) and existing contributors who helped make this release possible!
+
+## Intro to Radius
+
+If you're new to Radius, check out our website, [radapp.io](https://radapp.io), for more information. Also visit our [getting started guide](https://docs.radapp.io/getting-started/) to learn how to install Radius and create your first app.
+
+## Highlights
+
+<!-- TALK TO THE PM TEAM ABOUT WHAT HIGHLIGHTS TO ADD HERE -->
+
+TODO: PMs
```

Comment:
```suggestion
### `rad CLI` terminal
We have resolved a usability issue where the rad CLI prompt screen was truncated if the user's terminal window was small. The rad CLI prompt terminal now automatically scales to match the size of the user's terminal window.
```

Reply:
@willtsai/ @sylvainsf - I added a usability issue we fixed to the highlights. Anything else we want to summarize from the change log ?